apps/sms/views/shared/js/system_message_handler.js

```diff
@@ -329,6 +329,11 @@
       // Conversation view to the user, we remove that notification from the
       // tray that causes the second system message with "clicked" set to false.
       if (!notification.clicked || !notification.data) {
+        // Force close the app for notification removal case when app is hidden
+        // and navigation state is invalid(early reject while initializing).
+        if (document.hidden && !Navigation.isValidCurrentState()) {
+          return Promise.reject(window.close());
```

Comment:
If we don't close the app, the other alternative solution is finish the navigation init again or toPanel(inboxView).